src/euphorie/client/browser/templates/webhelpers.pt

```diff
@@ -351,13 +351,14 @@
   </metal:risk_identification>
 
   <metal:risk_evaluation define-macro="risk_evaluation">
-    <div class="risk-module pat-well notice pat-depends vertical evaluation"
+    <div class="risk-module pat-well notice pat-depends evaluation"
          id="evaluation"
          data-pat-depends="${view/evaluation_condition}; transition: slide"
          tal:define="
            evaluation_method python:risk.type in ('policy', 'top5') and risk.type or risk.evaluation_method;
+           skip_evaluation view/skip_evaluation;
          "
-         tal:condition="not:view/skip_evaluation"
+         tal:condition="python: not skip_evaluation and evaluation_method != 'policy'"
```

Comment:
This seems to be more than a markup change. Why is this necessary?

Reply:
Because for policy risks there is no evaluation and then we have an empty box with only a title that should not be there. Pointed out by Daniel as a "Schmutzeffekt", but actually needs a code change.

Reply:
Ah OK. I was afraid we might hide some logic change in a markup PR, but if it's about an empty box it should be fine.